Dockerfile

```diff
@@ -1,5 +1,5 @@
 # vim: set expandtab sw=4
-FROM golang:1.19-buster
+FROM golang:1.20.1-bullseye
```

Comment:
For the Go upgrade: Could you complete these [steps](https://github.com/web-platform-tests/wpt.fyi/blob/main/docs/upgrading-go.md)? This will allow the main tooling Dockerfile here to match the Go version deployed for 1) the webapp [[1](https://github.com/web-platform-tests/wpt.fyi/blob/main/webapp/web/app.yaml#L5)][[2](https://github.com/web-platform-tests/wpt.fyi/blob/main/webapp/web/app.staging.yaml#L5)] and 2) the [searchcache](https://github.com/web-platform-tests/wpt.fyi/blob/main/api/query/cache/service/Dockerfile).

Also, could you change this [line](https://github.com/web-platform-tests/wpt.fyi/blame/main/docs/upgrading-go.md#L18) to mention that we are using bullseye instead of buster.

Also, I see that you pinned the patch version 🎉 .
Could you update the wording [here](https://github.com/web-platform-tests/wpt.fyi/blob/main/webapp/web/app.staging.yaml#L5) and [here](https://github.com/web-platform-tests/wpt.fyi/blame/main/docs/upgrading-go.md#L22).

Reply:
Good catch! One question: in your last sentence you asked to updated the wording in 2 places, but the first link points to app.staging.yaml. Perhaps you meant [here](https://github.com/web-platform-tests/wpt.fyi/blame/main/docs/upgrading-go.md#L3)?

Reply:
Whoops! Thanks! That's the right link. Should read as: 

Could you update the wording [here](https://github.com/web-platform-tests/wpt.fyi/blame/main/docs/upgrading-go.md#L3) and [here](https://github.com/web-platform-tests/wpt.fyi/blame/main/docs/upgrading-go.md#L22).

Reply:
I made all of the proposed changes with the exception of [this one](https://github.com/web-platform-tests/wpt.fyi/blame/main/docs/upgrading-go.md#L22). It turns out that patch versions are not supported in the [go directive](https://go.dev/ref/mod#go-mod-file-go).